packages/core/admin/admin/src/translations/en.json

```
@@ -265,7 +265,7 @@
   "Settings.tokens.RegenerateDialog.title": "Regenerate token",
   "Settings.tokens.copy.editMessage": "For security reasons, you can only see your token once.",
   "Settings.tokens.copy.editTitle": "This token isn’t accessible anymore.",
-  "Settings.tokens.copy.lastWarning": "Make sure to copy this token, you won’t be able to see it again!",
+  "Settings.tokens.copy.lastWarning": "Make sure to copy this token.",
   "Settings.tokens.duration.30-days": "30 days",
```

Maybe this doesn't need to be as much of a warning now?

```suggestion
  "Settings.tokens.copy.message": "Copy this token for use elsewhere.",
```
I think "Settings.tokens.copy.editMessage" could be worded differently now too ?

---

yep this just follows the design changes
@maccomaccomaccomacco it's your time to shine

---

Another more general product question: what should we do in the case of a translation string change? In this case I think it makes sense to leave it slightly incorrect rather than removing them, but technically all the translations will be incorrect now.

---

yeah I was thinking about translations too, we'll need to translate to all the languages if we need it to be correct
I wish we had a script or something we can just run and it corrects the translations